### PR TITLE
[goreleaser] Setup automated builds for new file.d versions

### LINKED ISSUE #51: Feature: automated builds for new file.d releases
Automate the build and release process when new tags are created.
Binaries compiled for different platforms and architectures should be uploaded to the releases page along with .deb files and changelog.

### PR DESCRIPTION
**Pull request by ssnd**
closes #51

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -6,18 +6,47 @@ env:
   - CGO_ENABLED=0
 checksum:
   name_template: '{{ .ProjectName }}_checksums.txt'
-changelog:
-  skip: true
+# changelog:
+#   skip: true
```
Comment (anuriq) [inline .goreleaser.yml:10]: what does this do?

File: .goreleaser.yml [hunk 2]
```diff
@@ -6,18 +6,47 @@ env:
...
+    formats:
+      - deb
+    description: |- 
+      file.d is a tool for building data pipelines: 
+      read, process and output events.
+    homepage: https://ozontech.github.io/file.d/
+    vendor: Ozon Tech
+    bindir: /usr/bin
+    contents: 
+      - src: ./releaser/config.yaml
```
Comment (anuriq) [inline .goreleaser.yml:44]: `releaser` dir is best practice?

File: releaser/scripts/postinstall.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+install() {
+  pprint "file.d clean install..."
+  pprint "reload the service unit from disk..."
+  systemctl daemon-reload 
+  pprint "unmask the service..."
+  systemctl unmask file.d
+  pprint "enable file.d on system startup..."
+  systemctl enable file.d
+  systemctl restart file.d
```
Comment (anuriq) [inline releaser/scripts/postinstall.sh:15]: starting the daemon after installation is not best practice

**Review by anuriq — CHANGES_REQUESTED** [review]

**Review by anuriq — APPROVED** [review]

**Comment by anuriq** [comment]
LGTM

**Comment by vano144** [comment]
LGTM

**Comment by vano144** [comment]
LGTM

**Pull request merged**